AddOns/BasicFinancials/app/src/page/ContactAltAddressListBF.PageExt.al

```diff
@@ -0,0 +1,38 @@
+pageextension 20610 "Contact Alt. Address List BF" extends "Contact Alt. Address List"
+{
+    layout
+    {
+        modify("Company Name 2")
+        {
+            ApplicationArea = RelationshipMgmt, BFBasic;
+        }
+        modify("Address 2")
+        {
+            ApplicationArea = RelationshipMgmt, BFBasic;
+        }
+        modify("City")
+        {
+            ApplicationArea = RelationshipMgmt, BFBasic;
+        }
+        modify("Post Code")
+        {
+            ApplicationArea = RelationshipMgmt, BFBasic;
+        }
+        modify("County")
+        {
+            ApplicationArea = RelationshipMgmt, BFBasic;
+        }
+        modify("Country/Region Code")
+        {
+            ApplicationArea = RelationshipMgmt, BFBasic;
+        }
+        modify("Fax No.")
+        {
+            ApplicationArea = RelationshipMgmt, BFBasic;
+        }
+        modify("E-Mail")
```

Comment:
We don't hyphenate the word Email.

Reply:
If that's true, I don't understand why Microsoft have chosen to hyphenate the word Email in the following standard pages. 
I am only modifying fields that already exist in standard Business Central. The spelling of the fields has nothing to do with my tasks

page 5057 "Contact Alt. Address List", 
page 5116 "Salesperson/Purchaser Card"
page 5145 "Contact Through"

![image](https://user-images.githubusercontent.com/47662057/86133013-065a9480-bae8-11ea-9073-01c69df0176c.png)